SlevomatCodingStandard/Sniffs/Namespaces/UnusedUsesSniff.php

```diff
@@ -89,16 +92,18 @@ public function process(\PHP_CodeSniffer\Files\File $phpcsFile, $openTagPointer)
 
 					foreach ($annotations as $annotationName => $annotationsByName) {
 						if (preg_match('~^@(' . preg_quote($nameAsReferencedInFile, '~') . ')(?=[^a-z\\d]|$)~i', $annotationName, $matches)) {
-							$usedNames[$uniqueId] = true;
-
-							if ($matches[1] !== $nameAsReferencedInFile) {
-								/** @var \SlevomatCodingStandard\Helpers\Annotation $annotation */
-								foreach ($annotationsByName as $annotation) {
-									$phpcsFile->addError(sprintf(
-										'Case of reference name "%s" and use statement "%s" do not match.',
-										$matches[1],
-										$unusedNames[$uniqueId]->getNameAsReferencedInFile()
-									), $annotation->getPointer(), self::CODE_MISMATCHING_CASE);
+							if (!in_array($matches[1], $this->ignoredAnnotationNames, true)) {
```

Comment:
Is there a reason (I maybe missed something) why didn't you use the `$annotationName` variable? It can be used before the `preg_match` call.

Reply:
Please use `SniffSettingsHelper::normalizeArray`.